api/v2/api.go

```diff
@@ -615,6 +615,7 @@ func (api *API) setupRoutes(debug bool) error {
 	{
 		database.GET("/uploads", api.getUploadsForUser)
 		database.GET("/uploads/encrypted", api.getEncryptedUploadsForUser)
+		database.POST("/uploads/search", api.searchUploadsForUser)
```

Comment:
Why not use a GET for search too?

Reply:
we're using a post form here and previously when we tried a `GET` here there were some issues.